pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java

```diff
@@ -414,8 +425,9 @@ private int consumeFromWebSocket(String topic) {
                     LOG.debug("No message to consume after waiting for 5 seconds.");
                 } else {
                     try {
-                        System.out.println(Base64.getDecoder().decode(msg));
-                    }catch(Exception e) {
+                        String output = interpretByteArray(displayHex, Base64.getDecoder().decode(msg));
```

Comment:
this is the core of the change.
it is hidden inside the code reformat.

can you please revert the code reformat ?

Reply:
sure, I will fix it.